src/draw/lv_draw_buf.h

```diff
@@ -181,6 +181,20 @@ void lv_draw_buf_copy(void * dest_buf, uint32_t dest_w, uint32_t dest_h, const l
  */
 lv_draw_buf_t * lv_draw_buf_create(uint32_t w, uint32_t h, lv_color_format_t cf, uint32_t stride);
 
+/**
+ * Initialize a draw buf with the given buffer and parameters.
+ * @param draw_buf  the draw buf to initialize
+ * @param w         the buffer width in pixels
+ * @param h
+ * @param cf        the color format for image
+ * @param stride    the stride in bytes for image. Use 0 for automatic calculation.
+ * @param ptr       the buffer to use for image data. If NULL, a buffer will be allocated.
```

Comment:
`ptr` should be `data`.

The text "use for image data" is misleading I think, because we could misinterpret it with the image object. Better text would be: "_the buffer to use for the drawing data._"

Reply:
I think we should generally avoid using the term "image" in case of `lv_draw_buf`, because it is used for image objects (`lv_image`), and for me an image represents a whole image (decoded PNG or JPG, etc).

**It's just my opinion.**

I suggest to use the term "drawing", as the file name reflects that also: `lv_draw_buf`

Examples: "_color format for image_" -> "_color format for drawing_",  "_image data_" -> "_drawing data_", etc.

What do you think?

Reply:
I agree. I have updated to use `drawing` or not mention `image` at all. Please help to review it.